gzip.go

```diff
@@ -46,6 +46,17 @@ func ServeContent(w http.ResponseWriter, req *http.Request, name string, modTime
 		return
```

Comment:
This branch would reject if `gzip` is NOT mentioned, should we still do the `bz` check before we bail out? Otherwise, I think `Accept-Encoding: bz` would not work, as it doesn't contain `gzip` 🤔

Reply:
I've updated to find and serve static gzip files too, WDYT?